packages/keystone/lib/Keystone/index.js

```diff
@@ -658,13 +658,18 @@ module.exports = class Keystone {
           ...apps,
         ]
           .filter(({ prepareMiddleware } = {}) => !!prepareMiddleware)
-          .map(app =>
-            app.prepareMiddleware({
+          .map(app => {
+            const expressInstance = app.prepareMiddleware({
               keystone: this,
               dev,
               distDir: distDir || DEFAULT_DIST_DIR,
-            })
-          )
+            });
+
+            // Apply Express settings
+            this._expressOptions.forEach(([setting, value]) => expressInstance.set(setting, value));
```

Comment:
```suggestion
            Object.entries(expressOptions).forEach(([setting, value]) => expressInstance.set(setting, value));
```

Reply:
I can't do this since `expressOptions` is a ctor argument. It's used in `prepare` so I need to set that.